lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -48,7 +48,7 @@ class LNUrlBloc with AsyncActionsHandler {
       ])
           .where((l) => l != null)
           .map((l) => l.toLowerCase())
-          .where((l) => isLNURL(l))
+          .where((l) => isLNURL(l) || isLightningAddress(l))
```

Comment:
Did you try constructing the lnurl here and just pass it to the go library as the 'https' prefixed url?
If it is a valid lnurl then the go library will handle it well as is without any code changes.

Reply:
@roeierez since we don't want business logic in breezmobile, do you mean calling a separate fetchLNUrlFromAddress that will be in the Breez lib?

Reply:
AFAICT the we can build from the email the lnurl to fetch by convention (host + suffix). If we use this convention here we can use the regular lnurl endpoint in breez and not change breez library at all.
No sure it is required to create an endpoint only for this convention.

Reply:
This is business logic that will be expanded in the (near) future. I think this "construction" should be in Breez as well.

Reply:
Currently we anyway detect email by `isLightningAddress` so we know it is a lightning address. My take is that for this small convention logic right now a dedicated endpoint, protobuf etc is overkill.
Nevertheless I also don't think it is wrong.

Reply:
isLightningAddress is a simple regex for UI validation. It doesn't (and shouldn't) contain the URL construction logic. We shouldn't implement the Lightning address spec in Breez mobile. It's simple right now, but it's going to be expanded soon.

Reply:
Btw. if we construct the lnurl here, we lose the email address that was used and won't be able to display it to the user on the payment screen. Should I still push that change?

Reply:
@nochiel "payment screen", do you mean "payment details dialog"? What's the title of the payment for these payments (Zebedee etc)?

Reply:
> @nochiel "payment screen", do you mean "payment details dialog"? What's the title of the payment for these payments (Zebedee etc)?

Yes. That's it. 

Currently the title is the host of the lnurl e.g. zebedee.io/coinois.io, etc.

Reply:
I see. So, yes, I think it's important we show the address either in the title or in the payment details.

Reply:
> I see. So, yes, I think it's important we show the address either in the title or in the payment details.

I'm in favour of leaving the PR as is except if a bug or something I overlooked is found.